118.md

```diff
@@ -0,0 +1,92 @@
+# NIP-118
+
+## Polls
+
+`draft` `optional`
+
+This NIP defines the event scheme that describe Polls on nostr.
+
+The NIP makes the following tradeoffs.
+
+1. All votes are public.
+2. All votes are editable (latest response by timestamp considered last vote)
+
+## Events
+
+### Poll Event
+
+The poll event is defined as a `kind:1068` event. 
+
+- **content** key holds the label for the poll.
+
+Major tags in the poll event are:
+
+
+- **option**: The option tags contain an OptionId(any alphanumeric) field, followed by an option label field.
+- **relay**: One or multiple tags that the poll is expecting respondents to respond on.
+- **polltype**: can be "singlechoice", "multiplechoice", or "rankedchoice". Polls that do not have a polltype should be considered a "singlechoice" poll. 
+
+
+Example Event
+
+```json
+{
+  "content": "",
+  "created_at": 1719888496,
+  "id": "9d1b6b9562e66f2ecf35eb0a3c2decc736c47fddb13d6fb8f87185a153ea3634",
+  "kind": 1068,
+  "pubkey": "dee45a23c4f1d93f3a2043650c5081e4ac14a778e0acbef03de3768e4f81ac7b",
+  "sig": "7fa93bf3c430eaef784b0dacc217d3cd5eff1c520e7ef5d961381bc0f014dde6286618048d924808e54d1be03f2f2c2f0f8b5c9c2082a4480caf45a565ca9797",
+  "tags": [
+    ["label", "Pineapple on Pizza"],
```

Comment:
implemented NIP-88 on getwired.app
super easy to implement and works well. don't really see the point of the label tag on the poll event, but other than that no complaints

Reply:
it's an artifact from before the label was moved to the content field. Will update